### ISSUE TITLE
Unable to reset `DateTime` field to `null`

### ISSUE THREAD

**Issue by heymartinadams** [prisma/studio#632]
1. Prisma version (`prisma -v` or `npx prisma -v`):

`2.17.0`

2. Logs from Developer Tools Console or Command line, if any:

none

3. Does the issue persist even after updating to the latest `prisma` alpha? (`npm i -D @prisma/cli@dev`)

Yes.

4. Prisma schema (if relevant):

PostgreSQL database with AWS.

```prisma
model User {
  ...
  birthdate       DateTime?
  ...
}
```

https://user-images.githubusercontent.com/11673745/108134859-d0185800-706b-11eb-8a0c-df074bf5e7fc.mov

**Comment by applecat** [comment]
I have the same issue. When I'm trying to save a row with a nullable field - I see an error in the Studio's console: `TypeError: Cannot convert undefined or null to object`

1. Prisma version 
prisma               : 2.17.0
@prisma/client       : 2.17.0
Studio               : 0.353.0

2. Logs from Developer Tools Console or Command line, if any:

```
main.js:44 Uncaught (in promise) TypeError: Cannot convert undefined or null to object
    at Function.entries (<anonymous>)
    at r (main.js:44)
    at main.js:44
    at Array.map (<anonymous>)
    at r (main.js:44)
    at main.js:44
    at Array.map (<anonymous>)
    at r (main.js:44)
    at main.js:402
    at Array.forEach (<anonymous>)
    at Xt (main.js:402)
    at n.reduce.actions (main.js:402)
    at Array.reduce (<anonymous>)
    at nn (main.js:402)
    at Pn.commit (main.js:402)
    at we (main.js:16)
    at Pn.o (main.js:16)
    at onMatch (main.js:402)
    at main.js:44
    at c (main.js:370)
    at y.t._handleKey (main.js:370)
    at y.handleKey (main.js:370)
    at HTMLDocument.p (main.js:370)
    at HTMLDocument.o (main.js:402)
```

3. Does the issue persist even after updating to the latest prisma alpha? (npm i -D @prisma/cli@dev)
Yes

4. Prisma schema (if relevant):
```
model Company {
  ...
  logo      String?
  ...
}

**Comment by madebysid** [comment]
@heymartinadams @applecat I've managed to figure out what the issue was and have fixed this.

You should be able to update Prisma CLI to v2.18.0-dev.6 and see this resolved. `dev` versions are mostly for internal use though, and may contain unannounced breaking changes, so I'd recommend updating to the next stable version when that comes out.

Alternatively, you can use the desktop app, where this is fixed in the latest version (0.355.0). Let me know if it isn't.

Strangely enough, I didn't have a test that covered this, so I've added a few now. Thanks for reporting!
